src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -182,4 +163,55 @@ export class UBAClient {
 
     return { valid: true };
   }
+
+  protected async instantiateUBAFeeCalculator(chainId: number, token: string, fromBlock: number): Promise<void> {
+    const config = this.hubPoolClient.configStoreClient;
+    if (!isDefined(this.spokeUBAFeeCalculators[chainId]?.[token])) {
+      const spokeFeeCalculator = new UBAFeeSpokeCalculator(
+        chainId,
+        token,
+        this.getFlows(chainId, fromBlock),
+        0,
+        await config.getUBAFeeConfig(chainId, token)
+      );
+      this.spokeUBAFeeCalculators[chainId] = this.spokeUBAFeeCalculators[chainId] ?? {};
+      this.spokeUBAFeeCalculators[chainId][token] = spokeFeeCalculator;
+    }
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  protected async computeRealizedLpFee(
+    l1TokenAddress: string,
+    depositChainId: number,
+    refundChainId: number,
```

Comment:
Is it confirmed that the refundChain is relevant here? Per the UMIP, the `refund balancing fee` is handled as part of the relayerFeePct: https://github.com/UMAprotocol/UMIPs/blob/uba/UMIPs/umip-157.md#across-fees.

Reply:
Yes, we need it in the calculation of the utilization since we need both the spoke target balances for the deposit & refund chain.